### ISSUE TITLE
Documentation links are not translated

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#4860]
## What went wrong? 😲

When using the site in `fr` or `es`, when you click on the docs link in the LH nav, the docs site opens in `en`

## Expected behaviour 🤔
For the languages which we have translated docs, please open the docs in that language.
Simply append the language code to the docs root URL
e.g.

|  |  link       |
|:---|:-----------------|
|Bad: |   https://docs.msupply.foundation/docs/getting_started/dashboard/|
|Good:| https://docs.msupply.foundation/fr/docs/getting_started/dashboard/ |

Note the `/fr` after `.foundation`

## How to Reproduce 🔨

Steps to reproduce the behaviour:

## Your environment 🌱

- Open mSupply Version:
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

**Comment by mark-prins** [comment]
Release team: can we get this into v2.3 please? It would be a quick one to change and to test

**Comment by fergie-nz** [comment]
Hey Mark,

I'm wondering where the triggers are missing? From what I can see, internal navigation within the docs is maintaining languages. Navigating to the docs from somewhere where it is missing?

**Comment by roxy-dao** [comment]
If you are in Open mSupply and click the doc links while in a language other than English, it redirects you to the docs page in English, but ideally it should redirect you to the docs in the language selected in Open mSupply if available.

**Comment by mark-prins** [comment]
In `AppDrawer` we have
```
  const docsUrl = `${ExternalURL.PublicDocs}${
    EnvUtils.mapRoute(location.pathname).docs
  }`;
```

where the `PublicDocs` is in the enum:
```
export enum ExternalURL {
  PublicDocs = 'https://docs.msupply.foundation/docs',
}
```

we'd just need to replace `/docs` with `/fr/docs` if the current language is French or `/es/docs` if Spanish. I'd do that in AppDrawer myself

**Comment by fergie-nz** [comment]
Ah perfect! Thank you

**Comment by lache-melvin** [comment]
closed by #4903

**Comment by marthakeezy** [comment]
Tested in V2.3.1-RC1, passed, moving to Done ✅
